devel/ansible/roles/dev/files/.bashrc

```diff
@@ -9,6 +9,7 @@ fi
 # export SYSTEMD_PAGER=
 
 shopt -s expand_aliases
+alias bdiff-cover="btest; diff-cover /home/vagrant/bodhi/coverage.xml --compare-branch=develop --fail-under=100"
```

Comment:
diff-cover needs to be added to the ansible playbook too, so that the command is available